cellfinder_napari/tests/test_curation.py

```diff
@@ -29,3 +30,35 @@ def test_add_new_training_layers(curation_widget):
 
     assert layers[0].name == "Training data (cells)"
     assert layers[1].name == "Training data (non cells)"
+
+
+def test_cell_marking(curation_widget):
+    """
+    Check that marking cells and non-cells works as expected.
+    """
+    widget = curation_widget
```

Comment:
I noticed this in the above test. Just to check, this line has a use other than just cutting the number of characters in `curation_widget` Vs. `widget`?